code/_globalvars/misc.dm

```diff
@@ -16,3 +16,5 @@ GLOBAL_VAR_INIT(CHARGELEVEL, 0.001) // Cap for how fast cells charge, as a perce
 GLOBAL_LIST_EMPTY(powernets)
 
 GLOBAL_VAR_INIT(bsa_unlock, FALSE)	//BSA unlocked by head ID swipes
+
+GLOBAL_VAR_INIT(admin_delay_notice, "")

```

Comment:
`SSticker.delay_end` is the var that actually controls if the round end is delayed, you should maybe attach this to ssticker as well, since it seems odd to have one round end delayed var in global, and one in ssticker.